prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -377,19 +398,37 @@ class FHIRRouter(
         // default response
         val result = if (filter.isNullOrEmpty()) {
             defaultResponse
-        } else {
-            filter.all {
-                FhirPathUtils.evaluateCondition(
-                    CustomContext(bundle, bundle, shorthandLookupTable),
-                    bundle,
-                    bundle,
-                    replaceShorthand(it)
-                )
-            }
+        } else filter.all {
```

Comment:
Please add the {} back in for readability around the 'else' block for consistency.

Reply:
done